test/e2e/suite_test.go

```diff
@@ -22,8 +22,8 @@ func TestOperator(t *testing.T) {
 
 	ctx := context.Background()
 
-	//t.Run("CreateSingleDatacenterCluster", e2eTest(ctx, "single-dc", createSingleDatacenterCluster))
-	t.Run("CreateMultiDatacenterCluster", e2eTest(ctx, "multi-dc", createMultiDatacenterCluster))
+	t.Run("CreateSingleDatacenterCluster", e2eTest(ctx, "single-dc", createSingleDatacenterCluster))
```

Comment:
After thinking about it some more I do think it would be worthwhile to have some e2e test coverage for deployment Stargate on its own without a parent K8ssandraCluster. We still need a CassandraDatacenter though. It allows us to get the full test coverage of the Stargate controller without the complexity and overhead of the multi-cluster stuff. These tests would only require a single k8s cluster and could be run with the free tier GH Actions runners, which is a big plus.

wdyt?

Reply:
Of course, we should have no trouble running a multi-cluster env with only a single cluster with the free tier runner in GH Actions.